### PR TITLE
Feature/206

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
Change order of tables for correct execute refresh migration

issue: https://github.com/ProcessMaker/bpm/issues/206.

* The names of the tables were changed and the prefixes were eliminated, in addition the foreign keys were increased

* you need to evaluate if you still need the tables holidays, iso countries, sub division, locations, languages.

* The names of the tables that already have implementation were not changed since it is required to make more changes in the code (TIMER_EVENT, PROCESS_USER, EMAIL_EVENT, PROCESS_VARIABLES, APPLICATION, PROCESS_FILES).

* The definition and a refactor are required for tables APP_ * and MESSAGE_ *, only the order for the migration was modified.

### CODE REVIEW

**Comment by tdondich** [comment]
Why are there extra tables added now?

**Comment by marcoAntonioNina** [comment]
No extra table was added, only the order and foreign keys were modified.

File: database/migrations/2018_06_04_123400_create_users_table.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            $table->date('birthdate')->nullable();
+            $table->unsignedInteger('role_id')->nullable();
+            $table->string('time_zone')->nullable();
+            $table->string('lang')->nullable();
+            $table->dateTime('last_login')->nullable();
+
+            // setup relationship for Rol we belong to
+            $table->foreign('role_id')->references('id')->on('roles')->ondelete('cascade');
+
+            //check table
```
Comment (caleeli) [inline database/migrations/2018_06_04_123400_create_users_table.php]: Department table does not exists anymore, remove the following lines

File: database/migrations/2018_06_04_127500_create_report_table_columns_table.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            $table->integer('dynaform_id')->nullable()->unsigned();
+            $table->boolean('filter')->default(false);
+            $table->integer('process_variable_id')->nullable()->unsigned();
+
+            // Setup relationship for Additional tables we belong to
+            $table->foreign('report_table_id')->references('id')->on('additional_tables')->onDelete('CASCADE');
+            // Setup relationship for Form we belong to
+            $table->foreign('dynaform_id')->references('id')->on('forms')->onDelete('cascade');
+            // Setup relationship for Process Variable we belong to
+            //$table->foreign('process_variable_id')->references('VAR_ID')->on('PROCESS_VARIABLES')->onDelete('cascade');
```
Comment (caleeli) [inline database/migrations/2018_06_04_127500_create_report_table_columns_table.php]: Check this relationship

File: database/migrations/2018_06_04_123040_create_iso_sub_divisions_table.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        Schema::create('iso_sub_divisions', function(Blueprint $table)
+        {
+            $table->increments('id');
+            $table->unsignedInteger('country_id');
+            $table->string('uid', 2)->unique();
+            $table->string('name')->default('');
+
+            // setup relationship for Country we belong to
+            $table->foreign('country_id')->references('id')->on('iso_countries')->ondelete('cascade');
```
Comment (velkymx) [inline database/migrations/2018_06_04_123040_create_iso_sub_divisions_table.php]: iso_country_id

File: database/migrations/2018_06_04_123050_create_iso_locations_table.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        Schema::create('iso_locations', function(Blueprint $table)
+        {
+            $table->increments('id');
+            $table->string('uid', 5)->unique();
+            $table->unsignedInteger('country_id');
+            $table->string('name')->nullable();
+            $table->string('normal_name')->nullable();
+
+            // setup relationship for Country we belong to
+            $table->foreign('country_id')->references('id')->on('iso_countries')->ondelete('cascade');
```
Comment (velkymx) [inline database/migrations/2018_06_04_123050_create_iso_locations_table.php]: iso_country_id

File: database/migrations/2018_06_04_123050_create_iso_locations_table.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('iso_locations', function(Blueprint $table)
+        {
+            $table->increments('id');
+            $table->string('uid', 5)->unique();
+            $table->unsignedInteger('country_id');
```
Comment (velkymx) [inline database/migrations/2018_06_04_123050_create_iso_locations_table.php]: iso_country_id

File: database/migrations/2018_06_04_123040_create_iso_sub_divisions_table.php [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('iso_sub_divisions', function(Blueprint $table)
+        {
+            $table->increments('id');
+            $table->unsignedInteger('country_id');
```
Comment (velkymx) [inline database/migrations/2018_06_04_123040_create_iso_sub_divisions_table.php]: iso_country_id

File: database/migrations/2018_06_04_123100_create_roles_table.php [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('roles', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->unsignedInteger('parent_role_id')->nullable();
```
Comment (velkymx) [inline database/migrations/2018_06_04_123100_create_roles_table.php]: Is this a foreign key?

File: database/migrations/2018_06_04_124000_create_oauth_clients_table.php [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    public function up()
+    {
+        Schema::create('oauth_clients', function (Blueprint $table) {
+            $table->string('id')->primary();
+            $table->string('secret');
+            $table->string('name');
+            $table->string('description');
+            $table->string('website');
+            $table->string('redirect_uri');
+            $table->unsignedInteger('creator_user_id');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124000_create_oauth_clients_table.php]: change creator_user_id to user_id on this one. Unless there is another user_id.

File: database/migrations/2018_06_04_124100_create_oauth_scopes_table.php [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('oauth_scopes', function (Blueprint $table) {
+            $table->string('type');
+            $table->string('scope')->nullable();
+            $table->string('client_id')->nullable();
```
Comment (velkymx) [inline database/migrations/2018_06_04_124100_create_oauth_scopes_table.php]: oauth_client_id

File: database/migrations/2018_06_04_124100_create_oauth_scopes_table.php [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+            // setup relationship for Oauth Clients we belong to
+            $table->foreign('client_id')->references('id')->on('oauth_clients')->ondelete('cascade');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124100_create_oauth_scopes_table.php]: oauth_client_id

File: database/migrations/2018_06_04_124200_create_oauth_authorization_codes_table.php [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+        Schema::create('oauth_authorization_codes', function (Blueprint $table) {
+            $table->string('authorization_code')->primary();
+            $table->string('client_id');
+            $table->unsignedInteger('user_id');
+            $table->string('redirect_uri')->nullable();
+            $table->dateTime('expires');
+            $table->string('scope')->nullable();
+
+            // setup relationship for Oauth Client we belong to
+            $table->foreign('client_id')->references('id')->on('oauth_clients')->ondelete('cascade');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124200_create_oauth_authorization_codes_table.php]: oauth_client_id

File: database/migrations/2018_06_04_124300_create_oauth_access_tokens_table.php [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('oauth_access_tokens', function (Blueprint $table) {
+            $table->string('access_token')->primary();
+            $table->string('client_id');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124300_create_oauth_access_tokens_table.php]: oauth_client_id

File: database/migrations/2018_06_04_124300_create_oauth_access_tokens_table.php [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+            // If user_id is null, then this is client authentication
+            $table->unsignedInteger('user_id')->nullable();
+            $table->dateTime('expires');
+            $table->string('scope')->nullable();
+
+            // setup relationship for Oauth Client we belong to
+            $table->foreign('client_id')->references('id')->on('oauth_clients')->ondelete('cascade');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124300_create_oauth_access_tokens_table.php]: oauth_client_id

File: database/migrations/2018_06_04_124600_create_login_logs_table.php [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->unsignedInteger('user_id');
+            $table->string('status', 100)->default('');
+            $table->string('ip', 15)->default('');
+            $table->string('sid', 100)->default('');
+            $table->dateTime('init_date')->nullable();
+            $table->dateTime('end_date')->nullable();
+            $table->string('client_hostname', 100)->default('');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124600_create_login_logs_table.php]: Should this be nullable?

File: database/migrations/2018_06_04_124700_create_processes_table.php [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+            $table->integer('canceled_trigger_id')->nullable()->unsigned();
+            $table->integer('paused_trigger_id')->nullable()->unsigned();
+            $table->integer('reassigned_trigger_id')->nullable()->unsigned();
+            $table->integer('unpaused_trigger_id')->nullable()->unsigned();
+            $table->enum('visibility', ['PUBLIC', 'PRIVATE'])->default('PUBLIC');
+            $table->boolean('show_delegate')->default(true);
+            $table->boolean('show_dynaform')->default(0);
+            $table->integer('category_id')->nullable();
+            $table->timestamps();
+            $table->unsignedInteger('creator_user_id');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124700_create_processes_table.php]: user_id

File: database/migrations/2018_06_04_124700_create_processes_table.php [hunk 16]
```diff
@@ -0,0 +1,90 @@
...
+            $table->mediumText('original_source')->nullable()->default(null);
+
+            // Represents the BPMN file that represents this process, if one is available
+            $table->longText('bpmn')->nullable();
+
+            // setup relationship for Process we belong to
+            $table->foreign('parent_process_id')->references('id')->on('processes')->ondelete('cascade');
+
+            // setup relationship for User we belong to
+            $table->foreign('creator_user_id')->references('id')->on('users')->ondelete('cascade');
```
Comment (velkymx) [inline database/migrations/2018_06_04_124700_create_processes_table.php]: user_id

File: database/migrations/2018_06_04_125000_create_process_categories_table.php [hunk 17]
```diff
@@ -0,0 +1,34 @@
+<?php
```
Comment (velkymx) [inline database/migrations/2018_06_04_125000_create_process_categories_table.php]: Move this migration up. Is a requirement of processes

File: database/migrations/2018_06_04_125200_create_process_users_table.php [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('PROCESS_USER', function(Blueprint $table)
+        {
+            $table->uuid('PU_UID')->default('')->primary();
```
Comment (velkymx) [inline database/migrations/2018_06_04_125200_create_process_users_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_125300_create_process_variables_table.php [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('PROCESS_VARIABLES', function(Blueprint $table)
+        {
+            $table->increments('VAR_ID');
```
Comment (velkymx) [inline database/migrations/2018_06_04_125300_create_process_variables_table.php]: Update these variables to use the new style.

File: database/migrations/2018_06_04_125400_create_process_files_table.php [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+class CreateProcessFilesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('PROCESS_FILES', function(Blueprint $table)
```
Comment (velkymx) [inline database/migrations/2018_06_04_125400_create_process_files_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_125500_create_tasks_table.php [hunk 21]
```diff
@@ -43,14 +43,14 @@ public function up()
             /*************************************************************************************************
```
Comment (velkymx) [inline database/migrations/2018_06_04_125500_create_tasks_table.php]: is self_service_trigger_id the only trigger_id? if so rename to trigger_id

File: database/migrations/2018_06_04_125900_create_email_events_table.php [hunk 22]
```diff
@@ -0,0 +1,47 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('EMAIL_EVENT', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_125900_create_email_events_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_126500_create_script_tasks_table.php [hunk 23]
```diff
@@ -0,0 +1,40 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('script_tasks', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('tasks_id')->unsigned();
+            $table->integer('object_id')->unsigned();
```
Comment (velkymx) [inline database/migrations/2018_06_04_126500_create_script_tasks_table.php]: Is there an objects foreign key?

File: database/migrations/2018_06_04_126900_create_steps_table.php [hunk 24]
```diff
@@ -0,0 +1,48 @@
...
+     */
+    public function up()
+    {
+        Schema::create('steps', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('process_id')->unsigned();
+            $table->integer('task_id')->unsigned();
+
+            $table->integer('object_id')->unsigned();
```
Comment (velkymx) [inline database/migrations/2018_06_04_126900_create_steps_table.php]: Is there an objects table?

File: database/migrations/2018_06_04_127000_create_step_supervisors_table.php [hunk 25]
```diff
@@ -0,0 +1,43 @@
...
+     * @return void
+     */
+    public function up()
+    {
+
+        Schema::create('step_supervisors', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('step_id')->unsigned();
+            $table->integer('object_id')->unsigned();
```
Comment (velkymx) [inline database/migrations/2018_06_04_127000_create_step_supervisors_table.php]: Objects?

File: database/migrations/2018_06_04_127100_create_step_triggers_table.php [hunk 26]
```diff
@@ -0,0 +1,46 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('step_triggers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('step_id')->unsigned();
+            $table->integer('trigger_id')->unsigned();
+            $table->string('TYPE', 20)->default('');
```
Comment (velkymx) [inline database/migrations/2018_06_04_127100_create_step_triggers_table.php]: Is this an enum field?

File: database/migrations/2018_06_04_127200_create_sub_processes_table.php [hunk 27]
```diff
@@ -0,0 +1,56 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sub_processes', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('process_id')->unsigned();
+            $table->integer('task_id')->unsigned();
+            $table->integer('parent_process_id')->unsigned();
```
Comment (velkymx) [inline database/migrations/2018_06_04_127200_create_sub_processes_table.php]: How is the parent_process_id different from the process_id?

File: database/migrations/2018_06_04_127300_create_element_task_relations_table.php [hunk 28]
```diff
@@ -0,0 +1,44 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('element_task_relations', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('process_id')->unsigned();
+            $table->integer('task_id')->unsigned();
+            $table->integer('element_id')->unsigned();
```
Comment (velkymx) [inline database/migrations/2018_06_04_127300_create_element_task_relations_table.php]: Is there an elements table?

File: database/migrations/2018_06_04_127300_create_element_task_relations_table.php [hunk 29]
```diff
@@ -0,0 +1,44 @@
...
+            $table->string('element_type', 50)->default('');
```
Comment (velkymx) [inline database/migrations/2018_06_04_127300_create_element_task_relations_table.php]: Is this an enum?

File: database/migrations/2018_06_04_128000_create_applications_table.php [hunk 30]
```diff
@@ -0,0 +1,62 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APPLICATION', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_128000_create_applications_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_128100_create_configurations_table.php [hunk 31]
```diff
@@ -0,0 +1,46 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('configurations', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->string('object_uid', 128)->default('');
```
Comment (velkymx) [inline database/migrations/2018_06_04_128100_create_configurations_table.php]: Is this the same as the object_id?

File: database/migrations/2018_06_04_128200_create_sub_applications_table.php [hunk 32]
```diff
@@ -0,0 +1,47 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sub_applications', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('application_id')->nullable()->unsigned();
```
Comment (velkymx) [inline database/migrations/2018_06_04_128200_create_sub_applications_table.php]: Should this be nullable? Seems the application_id is required to create a sub_application (it is a sub after all)

File: database/migrations/2018_06_04_128400_create_app_folder_table.php [hunk 33]
```diff
@@ -0,0 +1,36 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_FOLDER', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_128400_create_app_folder_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_128400_create_app_folder_table.php [hunk 34]
```diff
@@ -0,0 +1,36 @@
...
+            $table->string('FOLDER_UID', 32)->default('')->primary();
+            $table->string('FOLDER_PARENT_UID', 32)->default('');
```
Comment (velkymx) [inline database/migrations/2018_06_04_128400_create_app_folder_table.php]: Index or Foreign Key?

File: database/migrations/2018_06_04_128500_create_app_document_table.php [hunk 35]
```diff
@@ -0,0 +1,58 @@
...
+class CreateAppDocumentTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_DOCUMENT', function(Blueprint $table)
```
Comment (velkymx) [inline database/migrations/2018_06_04_128500_create_app_document_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_128600_create_app_delay_table.php [hunk 36]
```diff
@@ -0,0 +1,57 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_DELAY', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_128600_create_app_delay_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_128700_create_app_event_table.php [hunk 37]
```diff
@@ -0,0 +1,43 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_EVENT', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_128700_create_app_event_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_128800_create_app_history_table.php [hunk 38]
```diff
@@ -0,0 +1,50 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_HISTORY', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_128800_create_app_history_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_128900_create_app_message_table.php [hunk 39]
```diff
@@ -3,7 +3,7 @@
 use Illuminate\Database\Migrations\Migration;
 use Illuminate\Database\Schema\Blueprint;
 
-class CreateAPPMESSAGETable extends Migration {
+class CreateAppMessagetable extends Migration {
```
Comment (velkymx) [inline database/migrations/2018_06_04_128900_create_app_message_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129000_create_app_notes_table.php [hunk 40]
```diff
@@ -0,0 +1,47 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_NOTES', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129000_create_app_notes_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129100_create_app_thread_table.php [hunk 41]
```diff
@@ -0,0 +1,41 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_THREAD', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129100_create_app_thread_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129200_create_app_timeout_action_executed_table.php [hunk 42]
```diff
@@ -0,0 +1,38 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_TIMEOUT_ACTION_EXECUTED', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129200_create_app_timeout_action_executed_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129300_create_message_application_table.php [hunk 43]
```diff
@@ -0,0 +1,50 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('MESSAGE_APPLICATION', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129300_create_message_application_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129400_create_message_event_definition_table.php [hunk 44]
```diff
@@ -0,0 +1,42 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('MESSAGE_EVENT_DEFINITION', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129400_create_message_event_definition_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129500_create_message_event_relation_table.php [hunk 45]
```diff
@@ -0,0 +1,39 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('MESSAGE_EVENT_RELATION', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129500_create_message_event_relation_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_129600_create_timer_event_table.php [hunk 46]
```diff
@@ -0,0 +1,49 @@
...
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('TIMER_EVENT', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_129600_create_timer_event_table.php]: Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.

File: database/migrations/2018_06_04_130000_update_various_tables.php [hunk 47]
```diff
@@ -0,0 +1,52 @@
...
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // Update process table to have foreign keys
+        Schema::table('processes', function (Blueprint $table) {
```
Comment (velkymx) [inline database/migrations/2018_06_04_130000_update_various_tables.php]: These should be in the processes migration

**Comment by velkymx** [comment]
Conflicts in build. Please merge dev and test. Then redo the PR

**Pull request closed without merging**